server/views/applications/pages/location-factors/describe-location-factors.njk

```diff
@@ -12,7 +12,7 @@
           classes: "govuk-label--m"
         },
         hint: {
-          html: '<p>For example SW1.</p>
+          html: '<p class="govuk-hint">For example SW1.</p>
```

Comment:
I wonder if this hint would be better below the rest of the copy. Probably a decision for @beth-dixxon